nilearn/image/image.py

```diff
@@ -778,3 +778,50 @@ def math_img(formula, **imgs):
         raise
 
     return new_img_like(niimg, result, niimg.get_affine())
+
+
+def load_niimg(niimg, ensure_ndim=None, atleast_4d=False, dtype=None,
+               wildcards=True):
+    """Load a Niimg-like object from filenames or list of filenames.
+
+    Parameters
+    ----------
+    niimg: Niimg-like object
```

Comment:
> I would say Nifti-like, parameter implies the same.

I am not sure that I agree. Niimg-like comes with a specific definition
in the nilearn context.